src/test_workflow/README.md

```diff
@@ -99,7 +99,65 @@ opensearch-dashboards=https://ci.opensearch.org/ci/dbc/bundle-build-dashboards/1
 
 ### Performance Tests
 
-TODO
+TODO: Add instructions on how run performance tests with `test.sh`
+
+
+
+#### How to identify regressions in performance tests
```

Comment:
Add to TOC, cleanup capitalization to match other topics.

I would rename to "Identifying Regressions in Performance Tests"